pyproject.toml

```diff
@@ -140,4 +162,4 @@ docstring-code-format = true
 #
 # This only has an effect when the `docstring-code-format` setting is
 # enabled.
-docstring-code-line-length = "dynamic"
+docstring-code-line-length = "dynamic"
```

Comment:
Please add new line

Reply:
yes, I added new line. please check.

Reply:
Hi, where should I add the new line? 
Do you mean I should add it as shown in the picture below?
![image](https://github.com/user-attachments/assets/8fe930c0-e717-41ad-bc10-05379e09adce)

Reply:
@korjsh That is right!